readthedocs/projects/forms.py

```diff
@@ -407,30 +407,31 @@ class Meta:
             # Basics and repo settings
             "name",
             "repo",
-            "repo_type",
-            "default_branch",
             "language",
-            "description",
-            # Project related settings
             "default_version",
             "privacy_level",
             "versioning_scheme",
-            "external_builds_enabled",
-            "external_builds_privacy_level",
+            "default_branch",
             "readthedocs_yaml_path",
-            "analytics_code",
-            "analytics_disabled",
-            "show_version_warning",
             # Meta data
             "programming_language",
             "project_url",
+            "description",
             "tags",
+            # Booleans
+            "external_builds_privacy_level",
+            "external_builds_enabled",
+            # Deprecated
+            "analytics_code",
+            "analytics_disabled",
+            "show_version_warning",
         )
 
+    # Make description smaller, only a CharField
     description = forms.CharField(
         required=False,
         max_length=150,
-        widget=forms.Textarea,
+        help_text=_("Short description of this project"),
```

Comment:
I think we still want the textarea widget here, otherwise the input will be pretty small. The length is fixed at 150 anyways.

Reply:
My goal here is to make the description much less prominent -- when it's a textarea, it's like 5x larger than all the other fields, and it's one of the least important. I think making it a normal input makes it more obvious that it also should be short :)